lib/rubocop/cop/rails/compact_blank.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks if collection can be blank-compacted with `compact_blank`.
+      #
+      # @example
+      #
+      #   # bad
+      #   collection.reject(&:blank?)
+      #   collection.reject(&:empty?)
+      #
+      #   # good
+      #   collection.compact_blank
+      #
+      #   # bad
+      #   collection.reject!(&:blank?)
+      #   collection.reject!(&:empty?)
```

Comment:
I'd through in the example with a Hash, since it's outstandingly bad.
```ruby
# bad
hashmap.reject { |_k, v| b.blank? }

# good
hashmap.compact_blank
```

Reply:
Sure! Thanks!